src/Container/MongoDbSnapshotAdapterFactory.php

```diff
@@ -91,8 +66,23 @@ public function packageName()
     /**
      * @inheritdoc
      */
-    public function containerId()
+    public function mandatoryOptions()
+    {
+        return ['snapshot_adapter' => ['options' => ['db_name']]];
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function defaultOptions()
     {
-        return 'snapshot_adapter';
+        return [
+            'snapshot_adapter' => [
+                'options' => [
+                    'snapshot_grid_fs_map' => [],
+                    'write_concern' => []
```

Comment:
This default overrides the default write concern in the adapter. You set it here to an empty array but the adapter checks for !== null